app/views/application/sitemap.xml.erb

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  <url>
+    <loc><%= "#{@host}"%></loc>
+    <xhtml:link rel="alternate" hreflang="en" href=<%= raw "\"#{@host}\""%>/>
+    <xhtml:link rel="alternate" hreflang="pt" href=<%= raw "\"#{@host}/pt/\""%>/>
+    <xhtml:link rel="alternate" hreflang="es" href=<%= raw "\"#{@host}/es/\""%>/>
+  </url>
+  <url>
+    <loc><%= "#{@host}/pt/"%></loc>
```

Comment:
Our site doesn't support Portuguese so I don't think we should include those links.  Either way, maybe have a list of supported languages and sitemap pages and use loops to generate the sitemap?

Reply:
You're right I was thinking of doing the loops as an update later but it makes sense to do at once. 
Portuguese was my bad. sorry. I was thinking of translating it but got no response about this idea so I will remove it.